src/lib/error.js

```diff
@@ -26,7 +26,12 @@ function ValidationError(message) {
   Object.defineProperty(this, 'name', { value: 'ValidationError' })
   Object.defineProperty(this, 'message', { value: getMessages(message).join('\n') })
   Object.defineProperty(this, 'collection', { value: message })
-  Error.captureStackTrace(this, ValidationError)
+
```

Comment:
Could you just add a comment here:
`// Fixes captureStackTrace missing on Safari`
Just let's us know why it's there 😄